db/migrate/20190827145628_create_join_table_sponsors_episodes.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableSponsorsEpisodes < ActiveRecord::Migration[5.2]
+  def change
+    create_join_table :sponsors, :episodes do |t|
+      # t.index [:sponsor_id, :episode_id]
```

Comment:
same here

Reply:
I just followed the guide :) Read about indexes, we won't use them in our project I guess